<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #DDDDDD;
    border: 0px solid #32414B;
    padding: 0px;
    color: black;
    selection-background-color: #1464A0;
    selection-color: #F0F0F0;
}
QLineEdit{
	background-color:white;
	color:black;
}

QProgressBar{
    background-color: #CFCFCF;
    border: 1px solid #32414B;
    color: #F0F0F0;
    border-radius: 4px;
	text-align:center;
}

QListWidget{
	background:#E9E9E9;
	color:white;
}
QListWidget:item:hover{
	background-color:#181818;
	border-left:4px slide #006eff
}
QComboBox{
	background-color:white;
}
QLCDNumber{
	border:1px solid white;
}
QPushButton:hover {
    border: 1px solid #19232D;
}
QPushButton:pressed {
    background-color: #6f6f6f;
	border: 1px solid #19232D;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Download video</string>
     </attribute>
     <widget class="QLineEdit" name="save_video">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Save Location</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="video_url">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Your URL</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_box">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Quality</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_bar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>401</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="video_save">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images/image.qrc">
        <normaloff>:/images/image/save.png</normaloff>:/images/image/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>61</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="video_start">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#EDEDED;
	color:black;
	border:1px solid black;
}
QPushButton:hover{
	background-color:#a8a8a8;
	border: 3px solid #148CD2;
}
QPushButton:pressed {
    background-color: #19232D;
    border: 1px solid #19232D;
}</string>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="video_search">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images/image.qrc">
        <normaloff>:/images/image/search.png</normaloff>:/images/image/search.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>61</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="shutdown_video">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Shutdown After Download</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="notification_video">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>make notification after download</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>download speed</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>64</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
	border:0px solid black;
}</string>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="list_video_download_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>141</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget:item{
	color:black;
}
QListWidget:item:hover{
	color:white;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Downloaded</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Download Playlist</string>
     </attribute>
     <widget class="QLineEdit" name="save_playlist">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Save Location</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playlist_url">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Your URL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Quality: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="playlist_start">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:#EDEDED;
	color:black;
	border:1px solid black;
}
QPushButton:hover{
	background-color:#a8a8a8;
	border: 3px solid #148CD2;
}
QPushButton:pressed {
    background-color: #19232D;
    border: 1px solid #19232D;
}</string>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progress_bar2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>401</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>181</y>
        <width>64</width>
        <height>20</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
	border:0px solid black;
}</string>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>playlist video</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>download speed</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>64</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
	border:0px solid black;
}</string>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="playlist_save">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images/image.qrc">
        <normaloff>:/images/image/save.png</normaloff>:/images/image/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>61</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="playlist_search">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images/image.qrc">
        <normaloff>:/images/image/search.png</normaloff>:/images/image/search.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>61</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="shutdown_playlist">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Shutdown After Download</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="notification_playlist">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>make notification after download</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="list_video_download">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>141</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget:item{
	color:black;
}
QListWidget:item:hover{
	color:white;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Video Downloaded</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>191</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#E9E9E9;</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>190</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="video_tab">
        <property name="toolTip">
         <string>video</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    border: 1px solid #19232D;
}
QPushButton:pressed {
    background-color: #6f6f6f;
	border: 1px solid #19232D;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images/image.qrc">
          <normaloff>:/images/image/youtube.png</normaloff>:/images/image/youtube.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>54</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playlist_tab">
        <property name="toolTip">
         <string>Playlist</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    border: 1px solid #19232D;
}
QPushButton:pressed {
    background-color: #6f6f6f;
	border: 1px solid #19232D;

}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images/image.qrc">
          <normaloff>:/images/image/playlist.png</normaloff>:/images/image/playlist.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>54</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Images/image.qrc"/>
 </resources>
 <connections/>
</ui>
